architecture/mandatory-elements.md

```diff
@@ -5,8 +5,7 @@ dateModified: 2023-03-28
 description: "Mandatory elements for all Government of Canada webpages."
 layout: default
 title: "Mandatory elements of the Canada.ca design"
----
-<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
+---<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
```

Comment:
This breaks the frontmatter, the DIV needs to be on its own line. This happened in a few files